src/licensedcode/data/rules/bsd-new_and_apache.yml

```diff
@@ -0,0 +1,3 @@
+licenses:
+    - bsd-new
+    - apache-2.0

```

Comment:
This is missing the `license_choice: yes` tag. Even though the text may look confusing this is really how you see a choice expressed